RELEASING/make_tarball.sh

```diff
@@ -22,8 +22,8 @@ usage() {
 }
 
 if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
-  if [ -z "${SUPERSET_VERSION}" ] || [ -z "${SUPERSET_RC}" ] || [ -z "${SUPERSET_PGP_FULLNAME}" ]; then
-    echo "No parameters found an no required environment variables set"
+  if [ -z "${SUPERSET_VERSION}" ] || [ -z "${SUPERSET_RC}" ] || [ -z "${SUPERSET_PGP_FULLNAME}" ] || [ -z "${SUPERSET_RELEASE_RC_TARBALL}" ]; then
+    echo "No parameters found and no required environment variables set"
```

Comment:
Since we are using regex to parse superset's version on `set_release_env` it would be nice to propagate this to all the other scripts